data/avro-schema/src/main/resources/avro/net/corda/data/membership/common/RegistrationStatus.avsc

```diff
@@ -4,7 +4,8 @@
   "namespace": "net.corda.data.membership.common",
   "doc": "Registration request status.",
   "symbols": [
-    "NEW",
+    "SUBMITTED",
+    "PENDING_MGM_NETWORK_ACCESS",
```

Comment:
Just for my own understanding, is it correct that `PENDING_MGM_NETWORK_ACCESS` means the registration has been sent to the MGM and we are now waiting for the MGM to grant access to the network?

Reply:
I think so; the member had received the request and sent it to the MGM to approve - I wonder if it's the best name...

Reply:
Perhaps `SUBMITTED` could be a good status for when it was sent to the MGM. Or else a `SENT_TO_MGM` status. 

I think as it stands, the members view of the status would be `PENDING_MGM_NETWORK_ACCESS` -> `PENDING_MEMBER_VERIFICATION` -> `PENDING_MANUAL_APPROVAL`/`PENDING_AUTO_APPROVAL` -> `APPROVED`/`DECLINED`. So there's a few more statuses after `PENDING_MGM_NETWORK_ACCESS` before actually getting access so it might be better to name it something more specific to indicate the request has been sent and we're waiting for the MGM to respond.